src/toil_scripts/exome_variant_pipeline/README.md

```diff
@@ -53,6 +36,23 @@ privileges you will need to build these tools from source, or bug a sysadmin abo
     1. Toil         pip install toil
     2. S3AM         pip install --pre s3am (optional, needed for uploading output to S3)
 
+## Installation
+
+If Toil is already installed globally (true for CGCloud users), or there are global dependencies (like Mesos),
+use virtualenv's `--system-site-packages` flag and *skip* running `pip install toil`.
+
+Type: `toil-exome` to get basic help menu and instructions
+
+To decrease the chance of versioning conflicts, install toil-scripts into a virtualenv: 
+
+- `virtualenv ~/toil-scripts` 
+- `source ~/toil-scripts/bin/activate`
+- `pip install toil`
+- `pip install toil-scripts`
+
+If Toil is already installed globally (true for CGCloud users), or there are global dependencies (like Mesos),
```

Comment:
Dupe of above. This whole section is ordered awkwardly. I would do it like that:

1) General installation instruction, with a `# CGCLoud users see below` note right next to the `pip install toil` step
2) Exceptions to the general instruction
3) Usage instructions